lib/chef/resource/windows_task.rb

```diff
@@ -59,6 +59,12 @@ class WindowsTask < Chef::Resource
       property :execution_time_limit, [String, Integer], default: "PT72H" # 72 hours in ISO8601 duration format
       property :minutes_duration, [String, Integer]
       property :minutes_interval, [String, Integer]
+      property :disallow_start_on_battery, [TrueClass, FalseClass], default: false,
+               introduced: "14.4",
+               description: "Scheduled task option when system is on AC power."
```

Comment:
This can get changed to "Start the task if the system is running on battery power"

Reply:
My suggestion for description based on :property name is `Disallow start of the task if the system is running on battery power.`